### PR TITLE
[enterprise-4.13] OADP-4657: Release Notes for 1.4.0 for OCP 4.13

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [OADP-4657](https://issues.redhat.com/browse/OADP-4657) - [DOC] Release Notes for OADP 1.4.0 for OCP 4.13
    * Release notes added to 4.13 to address the ACM issue. 

### Version(s):

* OCP 4.13

### Link to docs preview:

* [OADP 1.4.0 release notes in OCP 4.13](https://79411--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.html)

### QE review:
- [X] [QE has approved this change](https://github.com/openshift/openshift-docs/pull/79411#issuecomment-2273286724).

### CODE REVIEW

**Comment by amastbau** [comment]
LGTM

File: modules/oadp-1-4-0-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The Velero server has been updated from version 1.12 to 1.14. Note that, there are no changes in the Data Protection Application (DPA).
+
+This changes the following:
+
+* The `velero-plugin-for-csi` code is now available in the Velero code, which means no `init` container is needed for the plugin anymore.
+
+* Velero changed client Burst and QPS defaults from 30 and 20 to 100 and 100 respectively.
+
+* The `velero-plugin-for-aws` plugin updated default value of the `spec.config.checksumAlgorithm` field in BackupStorageLocations (BSLs) from `""` (no checksum calculation) to the `CRC32` algorithm. For more information, see link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/release-1.10/backupstoragelocation.md[Velero plugins for AWS Backup Storage Location]. The checksum algorithm types are known to work only with AWS.
```
Comment (adellape) [inline modules/oadp-1-4-0-release-notes.adoc:63]:
```suggestion
* The `velero-plugin-for-aws` plugin updated default value of the `spec.config.checksumAlgorithm` field in `BackupStorageLocation` objects (BSLs) from `""` (no checksum calculation) to the `CRC32` algorithm. For more information, see link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/release-1.10/backupstoragelocation.md[Velero plugins for AWS Backup Storage Location]. The checksum algorithm types are known to work only with AWS. 
```

File: modules/oadp-1-4-0-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+* Velero changed client Burst and QPS defaults from 30 and 20 to 100 and 100 respectively.
+
+* The `velero-plugin-for-aws` plugin updated default value of the `spec.config.checksumAlgorithm` field in BackupStorageLocations (BSLs) from `""` (no checksum calculation) to the `CRC32` algorithm. For more information, see link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/release-1.10/backupstoragelocation.md[Velero plugins for AWS Backup Storage Location]. The checksum algorithm types are known to work only with AWS. 
+Several S3 providers require the `md5sum` to be disabled by setting the checksum algorithm to `""`. Confirm `md5sum` algorithm support and configuration with your storage provider. 
++
+In  OADP 1.4, the default value for BSLs created within DPA for this configuration is `""`. This default value means that the `md5sum` is not checked, which is consistent with OADP 1.3. For BSLs created within DPA, update it by using the `spec.backupLocations[].velero.config.checksumAlgorithm` field in the DPA. If your BSLs are created outside DPA, you can update this configuration by using `spec.config.checksumAlgorithm` in the BSLs.
```
Comment (adellape) [inline modules/oadp-1-4-0-release-notes.adoc:66]:
```suggestion
In OADP 1.4, the default value for BSLs created within DPA for this configuration is `""`. This default value means that the `md5sum` is not checked, which is consistent with OADP 1.3. For BSLs created within DPA, update it by using the `spec.backupLocations[].velero.config.checksumAlgorithm` field in the DPA. If your BSLs are created outside DPA, you can update this configuration by using `spec.config.checksumAlgorithm` in the BSLs.
```

File: modules/oadp-backing-up-dpa-configuration-1-4-0.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+[id="oadp-backing-up-dpa-configuration-1-4-0_{context}"]
+= Backing up the DPA configuration
+
+You must back up your current `DataProtectionApplication` (DPA) configuration.
+
+.Procedure
+* Save your current DPA configuration by running the following command:
++
+.Example
```
Comment (adellape) [inline modules/oadp-backing-up-dpa-configuration-1-4-0.adoc:15]:
```suggestion
.Example command
```

File: modules/oadp-upgrading-oadp-operator-1-4-0.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+[id="oadp-upgrading-dpa-operator-1-4-0_{context}"]
+= Upgrading the OADP Operator
+
+Use the following procedure when upgrading the {oadp-first} Operator.
+
+.Procedure
+
+. Change your subscription channel for the OADP Operator from `stable-1.3` to `stable-1.4`.
+. Wait for the operator and containers to update and restart.
```
Comment (adellape) [inline modules/oadp-upgrading-oadp-operator-1-4-0.adoc:15]:
```suggestion
. Wait for the Operator and containers to update and restart.
```

**Comment by anarnold97** [comment]
@adellape - as I am just copying what was published for 4.14+, what is your advice, please?

1. Fix 4.13
1. Copy your comments to a new PR and push 4.13+

Thanks

**Comment by anarnold97** [comment]
Updating for all versions following review - SEE [PR](https://github.com/openshift/openshift-docs/pull/80253)

Removing the link was on my To-Do list

**Pull request closed without merging**